Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -393,8 +416,15 @@ def report_results(self, virtualdrive) -> int:
         os.makedirs(report_folder_path, exist_ok=True)
         #now parse the xml for errors
         failure_count = 0
-        logging.info("UnitTest Completed")
+        logging.info("UnitTest(s) Completed")
         for unit_test in self.test_list:
+            ignore_failure = False
+            if (os.path.basename(unit_test) in failure_exempt_tests.keys()):
+                now = datetime.datetime.now()
+                last_ignore_time = failure_exempt_tests[os.path.basename(unit_test)]
+                if (now - last_ignore_time).total_seconds() > FAILURE_EXEMPT_OMISSION_LENGTH:

```

Comment:
not sure i understand this.  Doesn't this read
if the difference between now and "last ignore time" is greater than OMISSION_LENGTH then ignore it?  
This seems like it would cause a failure to be ignored after 90 days. Don't we want the inverse?